Wikipedia/Code/EditSummaryViewController.swift

```diff
@@ -1,24 +1,33 @@
 
 import UIKit
 
-@objc protocol EditSummaryViewDelegate: AnyObject {
+protocol EditSummaryViewDelegate: AnyObject {
     func summaryChanged(newSummary: String)
     func learnMoreButtonTapped(sender: UIButton)
     func cannedButtonTapped(type: EditSummaryViewCannedButtonType)
 }
 
-@objc enum EditSummaryViewCannedButtonType: Int {
+// Int because we use `tag` from storyboard buttons.
+public enum EditSummaryViewCannedButtonType: Int {
     case typo, grammar, link
+    
+    var eventLoggingKey: String {
+        switch self {
+        case .typo:
+            return "typo"
+        case .grammar:
+            return "grammar"
+        case .link:
+            return "links"
+        }
+    }
 }
 
-@objcMembers class EditSummaryViewController: UIViewController, Themeable {
+class EditSummaryViewController: UIViewController, Themeable {
     static let maximumSummaryLength = 255
     
-    public var theme: Theme? {
+    public var theme: Theme = .standard {
         didSet {
-            guard let theme = theme else {
-                return
-            }
             apply(theme: theme)
```

Comment:
The `Themeable` protocol defines `applyTheme` so that function should set this theme variable as it does in other VCs

Reply:
Fixed!